Oscar Awards/Logic.java

```diff
@@ -0,0 +1,70 @@
+import java.io.*;
+public class Logic {
+    public static void main(String args[])
+    {
+        try  
+        {  
+            // String path="/home/nikhildewoolkar/Desktop/RedHat_Work/CSV_TO_JSON/the_oscar_award.csv";
+            File file=new File("the_oscar_award.csv");//path
+            FileReader fr=new FileReader(file);
+            BufferedReader br=new BufferedReader(fr);
+            String line;
+            String ans="[";
+            System.out.println("year_film,year_ceremony,ceremony,category,name,film,winner");
+            line=br.readLine();
+            while((line=br.readLine())!=null)  
+            {  
+                String s="[\"";
+                int commacount=0;
+                int i=0;
+                while(i<line.length())
+                {
+                    if(line.charAt(i)!=',' && line.charAt(i)!='\"')
```

Comment:
I would save `line.chartAt(i)` in a local variable to avoid multiple methods call

Reply:
I have saved line.charAt(i) in local variable. which resolves the issue.